third_party/miniz.c

```diff
@@ -2850,17 +2850,52 @@ void *tdefl_write_image_to_png_file_in_memory(const void *pImage, int w, int h,
   #include <sys/stat.h>
 
   #if defined(_MSC_VER) || defined(__MINGW64__)
+    #include <Windows.h>
+    #include <stringapiset.h>
+    #include <wchar.h>
+    
+    // return NULL if inStr is NULL or if the conversion failed
```

Comment:
a note about who is responsible for de-allocating the memory returned by this function is important.